PERFORMANCE.md

```diff
@@ -1,100 +1,287 @@
-It takes much less time to complete using a colocated orchestrator since all the computations are happening on the same node. Less time it takes to communicate since the db and app are on the same node.
+The following are standard deployment scaling results from the cpp-throughput.
+
+#### Throughput Std Run Configuration File
+```bash
+[run]
+name = run-2023-07-05-21:26:18
+path = results/throughput-standard-scaling/run-2023-07-05-21:26:18
+smartsim_version = 0.4.2
+smartredis_version = 0.3.1
+db = redis-server
+date = 2023-07-05
+language = ['cpp']
+
+[attributes]
+colocated = 0
+client_per_node = [48]
+client_nodes = [4, 8, 16, 32, 64, 128]
+database_nodes = [4, 8, 16]
+database_cpus = [8]
+iterations = 100
+tensor_bytes = [1024, 8192, 16384, 32769, 65538, 131076, 262152, 524304, 1024000]
+language = ['cpp']
+wall_time = 05:00:00
+```
+
+#### Put Tensor (send image to database) & Unpack Tensor (retrieve the image)
+![Throughput Std Unpack](/figures/new_std_thro.png "Throughput Standard")
 
-## Data Aggregation Standard
+## Throughput Colocated
 
-The following are scaling results for a data aggregation test, run with 32 client threads on each node. The databases were run on 36 CPUs and 4 threads per dataset. 
+The following are colocated deployment scaling results from the cpp-throughput.
+
+#### Throughput Colo Run Configuration File
+
+```bash
+[run]
+[run]
+name = run-2023-08-20-21:03:55
+path = results/throughput-colocated-scaling/run-2023-08-20-21:03:55
+smartsim_version = 0.5.0
+smartredis_version = 0.3.1
+db = redis-server
+date = 2023-08-20
+language = ['cpp']
+
+[attributes]
+colocated = 1
+custom_pinning = [False]
+client_per_node = [32]
+client_nodes = [16, 32, 64, 128]
+database_cpus = [8]
+iterations = 100
+tensor_bytes = [1024]
+language = ['cpp']
+```
+
+#### Put Tensor (send image to database) & Unpack Tensor (retrieve the image)
+![Throughput colo Unpack](/figures/test.png "Colocated Throughput")
 
-#### Get List - retrieve the data from the aggregation list
-![Data Agg Get List](/figures/get_list_data_agg.png "Data Aggregation Standard")
+## Throughput Performance Analysis
 
-## Data Aggregation Standard Py
-The following are scaling results for a data aggregation py test, run with 32 client threads on each node. The databases were run on 36 CPUs and 4 threads per dataset.
+In this section, we will compare throughput clients: `put-tensor` and `unpack-tensor`,
+for colocated and standard deployment.
+
+> Note that Throughput measures the total time it takes to push and pull data from a database.
+The SmartSim Scaling studies produces a series of generated tensors to add and pull from a Redis Database.
+
+- `put_tensor` : We notice that for both colocated and standard deployment, put_tensor performance
+extremley quickly with both medians performing faster than .001 seconds. The difference here lies 
+within the outside points. Looking at the standard violin plots, the outside point values are much
```

Comment:
Outside points? You mean the high end of the distribution?

Reply:
yes I got outside points from the internet merp